jdaviz/configs/imviz/plugins/tools.py

```diff
@@ -122,10 +122,6 @@ def deactivate(self):
     def on_click(self, data):
         self.viewer.blink_once(reversed=data['event']=='contextmenu')  # noqa: E225
 
-        # Also update the coordinates display.
-        data['event'] = 'mousemove'
```

Comment:
This was here so blink without moving cursor would update. Is this still the case?

Reply:
Yes, this is handled by the `_layers_changed` callback method now